docker-compose.yml

```diff
@@ -1,7 +1,25 @@
-version: '3'
-
+---
+version: "3.8"
 services:
-  app:
-    build: ./
+  puter:
+    container_name: puter
+    image: ghcr.io/heyputer/puter:latest
+    pull_policy: always
+    # build: ./
+    restart: unless-stopped
     ports:
-      - 4000:4000    
+      - '4100:4100'
+    environment:
+      # TZ: Europe/Paris
+      # CONFIG_PATH: /etc/puter
+      PUID: 1000
+      PGID: 1000
+    volumes:
+      - ./config:/opt/puter/app/volatile/config
+      - ./data:/opt/puter/app/volatile/runtime
+    # healthcheck:
+    #   test: wget --no-verbose --tries=1 --spider http://localhost:4100 || exit 1
+    #   interval: 5m
+    #   timeout: 3s
+    #   retries: 3
+    #   start_period: 2m
```

Comment:
```suggestion
    healthcheck:
      test: wget --no-verbose --tries=1 --spider http://puter.localhost:4100/test || exit 1
      interval: 5m
      timeout: 3s
      retries: 3
      start_period: 2m
```

Reply:
Happy to commit that, although have you tested it? Otherwise I'll get to it tomorrow morning my time (CEST).

Easiest way to check is to run that wget command from inside the container as it is running.
Open a new shell and run:
```
docker ps
docker exec -it <container id or name from "docker ps"> ash
```
We need to use `ash` as shell instead of `bash`, as the Node base image for this puter container image is built on Alpine, which uses `ash` as its main shell.

Reply:
I tested it, it works perfecty:
```$ docker ps
CONTAINER ID   IMAGE     COMMAND                  CREATED         STATUS                   PORTS                    NAMES
4736a94580a3   puter     "docker-entrypoint.s…"   3 minutes ago   Up 3 minutes (healthy)   0.0.0.0:4100->4100/tcp   unruffled_solomon```
Notice the `(healthy)` in the `STATUS` column.

Thanks @KernelDeimos !